### PR TITLE
Resolved issue #2407

### PR DESCRIPTION
**Pull request by sainiak009**
Please review it.
Added required fields in marshaling for export and import should work fine.

### CODE REVIEW

File: app/api/events.py [hunk 1]
```diff
@@ -98,7 +98,9 @@
     'social_links': fields.List(fields.Nested(SOCIAL_LINK), attribute='social_link'),
     'call_for_papers': fields.Nested(EVENT_CFS, allow_null=True),
     'version': fields.Nested(EVENT_VERSION),
-    'has_session_speakers': fields.Boolean(default=False)
+    'has_session_speakers': fields.Boolean(default=False),
+    'thumbnail' : fields.String(),
```
Comment (niranjan94) [inline app/api/events.py]: Instead of `fields.String()` you could use `fields.Uri()` or `fields.ImageUri()` for proper data validation.

**Comment by niranjan94** [comment]
@sainiak009 good job with the contributions :) ... Just a small tip, the commit messages could be more descriptive about the change that was made in it. Makes the commit history easier to read ...

File: app/api/events.py (hunk 1 above)
Comment (sainiak009) [inline app/api/events.py]: Changed fields.String() to fields.Uri() for thumbnail and large fields.

**Comment by sainiak009** [comment]
Changes are done. Please review

**Pull request merged**